src/Fixer/Basic/EmptyBodyFixer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Basic;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class EmptyBodyFixer extends AbstractFixer
```

Comment:
I don't like `empty_body` name. @keradus once said fixers should be named from desired state (after fix), which here is "single-line empty body", not "empty body" itself. It could be just `empty_body` if it also handled comments within braces (extract them outside of the body), which is not the case by looking at the tests.

Reply:
"single-line empty body" would also be true for a body like `{ return true; }` or `{ /* @todo: implement this */ }`.

By empty body, I meant literally empty, like nothing between `{` and `}`. Would `abbreviated_as_pair_of_braces_and_placed_on_the_same_line_as_the_previous_symbol_and_separated_by_a_space` be a better name? 😉

Reply:
> "single-line empty body" would also be true for a body like { return true; } or { /* @todo: implement this */ }.

No, because those bodies are not _empty_ 😉.

On the other hand, you can have:

```php
class Foo {
    public function __construct()
    {




    }
}
```

and the body _is_ empty (so it fits fixer's current name), but is _not_ single line. So if desired state is single-line empty body (`{}`), it should be named `single_line_empty_body`.